cli/module_registry.go

```diff
@@ -64,11 +64,11 @@ func CreateModuleCommand(c *cli.Context) error {
 		return err
 	}
 	if org == nil {
-		return errors.Errorf("Unable to determine org from orgID(%q) and namespace(%q)", orgIDArg, publicNamespaceArg)
+		return errors.Errorf("unable to determine org from orgID(%q) and namespace(%q)", orgIDArg, publicNamespaceArg)
 	}
 	// Check to make sure the user doesn't accidentally overwrite a module manifest
 	if _, err := os.Stat(defaultManifestFilename); err == nil {
-		return errors.New("Another module's meta.json already exists in the current directory. Delete it and try again")
+		return errors.New("another module's meta.json already exists in the current directory. delete it and try again")
```

Comment:
Ooh, for my edification when writing the procedural for this: how do we determine that a detected `meta.json` file is for a different module than the one we're `create` | `update` | `upload` ing? The initial blank one generated by `create` doesn't appear to have any unique characteristics that I could envision powering such a determination.

Reply:
It looks like a requirement of creating a module is that no `meta.json` exists in the current directory so that a new one can be created later in `writeManifest`. I think it's assumed here that if one exists it belongs to another module? @zaporter-work probably knows more.

Reply:
The assumption is that if you are creating a module and there is already a `meta.json`, it is an error case that the user should investigate. 

It would be possible to do intelligent behavior like reading the `meta.json` and checking if it matches the module the user is trying to create, but the extra complexity did not seem like it would  produce a much better error.

Reply:
In this case, it felt like a less clever error would be easier for a user to understand.

Reply:
Absolutely makes sense, just want to be sure I draft the docs accordingly -- thanks for this clarification! Expect a PR review request Monday 😅